cluster/images/etcd/migrate/data_dir.go

```diff
@@ -88,7 +87,7 @@ func (d *DataDirectory) Backup() error {
 	if err != nil {
 		return err
 	}
-	err = exec.Command("cp", "-r", d.path, backupDir).Run()
+	err = copyDirectory(d.path, backupDir)
```

Comment:
Sorry to sound in late on this. The files are usually accessed via a `volumeMount` (https://github.com/kubernetes/kubernetes/blob/3446ffbb4a5328e88a75afc36a258c1f7adb1b89/cluster/gce/manifests/etcd.manifest#L77), but it would be odd for the volumeMount to allow for the files under it to span multiple filesystems/os mount points, so I think this is fine.

Reply:
great!